### ISSUE TITLE
Some problems about wrap a sliver widget

### ISSUE THREAD

**Issue by Liuzhiyang94** [mchome/flutter_advanced_networkimage#85]
When I wrap a sliver widget in the `ZoomableWidget`,I found that I can't scroll it.So I go to the source code, and add a `Listener()` widget contains the `GestureDetector`.But it's too hard to handle the clash between sliver widget and `onScaleUpdate` .

**Comment by mchome** [comment]
Oh, if you are using some kind of scrollable widgets in `ZoomableWidget`, the events of these widgets will be ignored.

**Comment by Liuzhiyang94** [comment]
> Oh, if you are using some kind of scrollable widgets in `ZoomableWidget`, the events of these widgets will be ignored.

Yes.Is there any way to handle this?

**Comment by mchome** [comment]
No, i have no ideas with this. I built the `ZoomableList` with `GestureDetector` and `Column`, you will need to implement your own one for your needs.

**Comment by mchome** [comment]
Ref: https://github.com/flutter/flutter/issues/15011
